test/integration/setup.js

```diff
@@ -511,7 +511,7 @@ async function setup() {
     productId: 0,
     weight: 100,
     initialPrice: 1000,
-    targetPrice: 1000,
+    targetPrice: 100,
```

Comment:
This resolved a broken test in `integration/Pool/token-price.js` 
An assertion was off at line 266 `    expect(spotTokenPricePostMCRPosting).to.be.lt(spotTokenPricePreMCRPosting);`
I'm not exactly sure why it effected this. I was only adjusting it because I noticed the initial prices (cover products) were lower than the target price.